gno.land/cmd/gnoland/config_get.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/gnolang/gno/tm2/pkg/bft/config"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+var errInvalidConfigGetArgs = errors.New("invalid number of config get arguments provided")
+
+// newConfigGetCmd creates the config get command
+func newConfigGetCmd(io commands.IO) *commands.Command {
+	cfg := &configCfg{}
+
+	cmd := commands.NewCommand(
+		commands.Metadata{
+			Name:       "get",
+			ShortUsage: "config get <key>",
+			ShortHelp:  "Shows the Gno node configuration",
```

Comment:
btw, Hariom changed all these short helps to not be capitalised in #1701, and I agree, so I suggest you do the same

Reply:
I went ahead and swapped all short helps in the `gnoland` suite, before realizing Hariom did the same in the PR you've linked:

[7412856](https://github.com/gnolang/gno/pull/1605/commits/74128567133aae24e37c2c7f99fc8081d1b6b22f)